server/odc-service/src/main/java/com/oceanbase/odc/metadb/collaboration/EnvironmentRepository.java

```diff
@@ -39,4 +40,9 @@ public interface EnvironmentRepository extends JpaRepository<EnvironmentEntity,
     @Transactional
     int updateEnabledById(@Param("id") Long id, @Param("enabled") Boolean enabled);
 
+    @Query(value = "select name, style from collaboration_environment where id in (:ids)", nativeQuery = true)
```

Comment:
no need to add this native query sql

Reply:
don' need any of these three annotations? here is reference `update`  operation. why no need we use query annotation here?

Reply:
No, you don't. Please read the Spring Data JPA docs before writing the code if you do not actually understand the meaning of these three annotations.

Reply:
Right, `EnvironmentRepository` extend JPA. I refer to `update`. I don't have an in-depth understanding of the differences here.

Reply:
if don't use `@Query`,  Similar to MybatisPlus, got it.